dojo/models.py

```diff
@@ -2185,6 +2185,8 @@ def hash_code_fields(self):
             elif (self.scan_type in settings.HASHCODE_FIELDS_PER_SCANNER):
                 deduplicationLogger.debug(f"using HASHCODE_FIELDS_PER_SCANNER for scan_type: {self.scan_type}")
                 hashCodeFields = settings.HASHCODE_FIELDS_PER_SCANNER[self.scan_type]
+            else:
+                deduplicationLogger.debug(f"test_type name {self.test_type.name} and scan_type {self.scan_type} not found in HASHCODE_FIELDS_PER_SCANNER")
```

Comment:
```suggestion
                deduplicationLogger.warning(f"test_type name {self.test_type.name} and scan_type {self.scan_type} not found in HASHCODE_FIELDS_PER_SCANNER")
```
I recently did something like this too, but was bumping to the severity to warning to make it more noticeable in logs to make something optimal
https://github.com/DefectDojo/django-DefectDojo/pull/11566/files

Reply:
ok, let's hope it doesn't get too noisy :-)